langwatch/src/server/posthog.ts

```diff
@@ -0,0 +1,20 @@
+import { PostHog } from "posthog-node";
+import { env } from "../env.mjs";
+
+export const posthog = env.POSTHOG_KEY
```

Comment:
Consider exporting only a `getPostHogInstance` that makes it clear that posthog might be null and also that it's immutable.